test/Chainweb/Test/Pact/Checkpointer.hs

```diff
@@ -103,6 +110,37 @@ defModule idx = [text| ;;
 tIntList :: [Int] -> Term n
 tIntList = toTList (TyPrim TyInteger) def . map toTerm
 
+testModuleName :: TestTree
+testModuleName = withResource initializeSQLite freeSQLiteResource $
+  runSQLite' (testCase "testModuleName" . go)
+  where
+    go res = do
+
+      (CheckpointEnv {..}, SQLiteEnv {..}) <- res
+
+      -- init genesis
+      let hash00 = nullBlockHash
+      void $ _cpRestore _cpeCheckpointer Nothing
+      _cpSave _cpeCheckpointer hash00
+
+      -- block 1: write module records
+      hash01 <- BlockHash <$> liftIO (merkleLogHash "0000000000000000000000000000001a")
```

Comment:
There is also an arbitrary instance for `BlockHash` that can be used to get arbitrary block hashes.

For random arbitrary values

```Haskell
hash01 <- generate
```

For deterministic arbitrary values:

```Haskell
import Test.QuickCheck.Gen
import Test.QuickCheck.Random

getArbitrary :: Arbitrary a => Int -> a
getArbitrary seed = unGen arbitrary (mkQCGen 0) seed

-- ...

let hash01 = getArbitrary @BlockHash 0
```

Reply:
Nice, added this to Chainweb.Test.Utils